release/device-mqtt-go.yaml

```diff
@@ -0,0 +1,20 @@
+---
+name: 'device-mqtt-go'
+version: '2.1.0'
+releaseName: 'jakarta'
+releaseStream: 'main'
+lts: true
+commitId: 'f142b7d4d9dc0d163bb8d716c4eb7ddee57de476'
```

Comment:
Waiting on last issue to be implemented and merged. 
https://github.com/edgexfoundry/device-mqtt-go/issues/321